mainnet/collections.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "jobIDs": [1, 2, 3, 5, 6, 7, 9, 10],
+    "jobIDs": [1, 2, 3, 6, 7, 9, 10],
```

Comment:
If we're strictly avoiding stable coins/wrapped tokens, wouldn't it make more sense to remove the job itself than changing the collections only?